integreat_compass/cms/templates/_base.html

```diff
@@ -1,3 +1,8 @@
+<script>
+    setTimeout(function() {
+        document.getElementById('messages').setAttribute('style', 'display:none !important');
+    }, 3000);
+</script>
```

Comment:
If we do want this (personally I don't really see the need), please move it into an appropriate file.

Reply:
I think this is the appropriate file, where do you think this should be?

If we remove it, we will have multiple success messages displayed at once, when a user approves multiple offers.

Reply:
Probably @charludo means to put the JS code into a separate `.ts` file and include as part of the webpack bundling process instead of putting it into the html template.

Reply:
Also, you could add the class `hidden` instead of setting the style attribute.

Reply:
Yes, sorry, I meant moving it to its own file like Timo said.

> If we remove it, we will have multiple success messages displayed at once, when a user approves multiple offers.

That still happens if they do so in quick succession. There's other parts around the project as well where multiple error/success messages are shown. Maybe you could make the messages unique (so they do not look like accidental multiples) by changing them to something like `Your vote to approve "TItle of the voted offer" has been saved.`?